db/structure.sql

```diff
@@ -9,6 +9,13 @@ SET xmloption = content;
 SET client_min_messages = warning;
 SET row_security = off;
 
+--
```

Comment:
this seems like an incorrect addition. Please re-concile structure.sql with master